src/storage/web-api.ts

```diff
@@ -23,6 +23,15 @@ import type { StorageValue } from './types.js';
  */
 export function buildWebStorage(storage: Storage, prefix = 'axios-cache-') {
   return buildStorage({
+    clear: () => {
+      const keys = Object.keys(storage);
+      for (const key of keys) {
```

Comment:
`for...in`

Reply:
`for...in` would work yes, that's a good point.
As for prefix - it is always defined because it has default value defined here:
https://github.com/arthurfiorette/axios-cache-interceptor/blob/934f345848126c15b77e7bfc81203ea681bd6717/src/storage/web-api.ts#L24

My understanding is - removing all items using prefix is safe, unless users deliberately sabotage their own storage by naming outside keys the same as prefix. So it is essentially the same as remove method but for all of them:
https://github.com/arthurfiorette/axios-cache-interceptor/blob/934f345848126c15b77e7bfc81203ea681bd6717/src/storage/web-api.ts#L40-L42

Am I missing some edge case where effect can leak outside the storage?

Reply:
sounds good